api/src/pcapi/tasks/gdpr_tasks.py

```diff
@@ -0,0 +1,249 @@
+    actions_history = (
+        history_models.ActionHistory.query.filter(
+            history_models.ActionHistory.user == user,
+            history_models.ActionHistory.actionType.in_(
+                [
+                    history_models.ActionType.USER_SUSPENDED,
+                    history_models.ActionType.USER_UNSUSPENDED,
+                    history_models.ActionType.USER_PHONE_VALIDATED,
+                    history_models.ActionType.USER_EMAIL_VALIDATED,
+                ],
+            ),
+        )
+        .order_by(history_models.ActionHistory.id)
+        .all()
+    )
+    return [serializers.ActionHistory.from_orm(a) for a in actions_history]
+
+
+def _extract_beneficiary_valisation(user: users_models.User) -> list[serializers.BeneficiaryValidation]:
+    beneficiary_fraud_checks = (
+        fraud_models.BeneficiaryFraudCheck.query.filter(fraud_models.BeneficiaryFraudCheck.user == user)
+        .order_by(fraud_models.BeneficiaryFraudCheck.id)
+        .all()
+    )
+    return [serializers.BeneficiaryValidation.from_orm(b) for b in beneficiary_fraud_checks]
+
+
+def _extract_booking_data(user: users_models.User) -> list[serializers.Booking]:
+    bookings_data = (
+        bookings_models.Booking.query.filter(
+            bookings_models.Booking.user == user,
+        )
+        .options(
+            joinedload(bookings_models.Booking.stock).joinedload(offers_models.Stock.offer),
+            joinedload(bookings_models.Booking.venue),
+            joinedload(bookings_models.Booking.venue).joinedload(offerers_models.Venue.managingOfferer),
+        )
+        .order_by(bookings_models.Booking.id)
+    )
+    bookings = []
+    for booking_data in bookings_data:
+        offerer = booking_data.venue.managingOfferer
+        bookings.append(
+            serializers.Booking(
+                cancellationDate=booking_data.cancellationDate,
+                dateCreated=booking_data.dateCreated,
+                dateUsed=booking_data.dateUsed,
+                quantity=booking_data.quantity,
+                amount=booking_data.amount,
+                status=booking_data.status,
+                name=booking_data.stock.offer.name,
+                venue=booking_data.venue.publicName or booking_data.venue.name,
+                offerer=offerer.name,
+            )
+        )
+    return bookings
+
+
+def _extract_brevo_data(user: users_models.User) -> dict:
+    return get_raw_contact_data(user.email)
+
+
+def _get_now_as_tuple() -> tuple[int, int, int, int, int, int]:
+    now = datetime.utcnow()
+    return (now.year, now.month, now.day, now.hour, now.minute, now.second)
+
+
+def _dump_as_json_bytes(container: serializers.DataContainer) -> tuple[zipfile.ZipInfo, bytes]:
+    json_bytes = container.json(indent=4).encode("utf-8")
+    file_info = zipfile.ZipInfo(
+        filename=f"{container.internal.user.firstName}_{container.internal.user.lastName}.json",
+        date_time=_get_now_as_tuple(),
```

Comment:
[détail / question de goût]

```suggestion
        date_time=datetime.datetime.utcnow().timetuple()[:6],
```